### PR TITLE
add gokart.testing.assert_frame_equal

### PR DESCRIPTION
**Pull request by mski-iksm**
I've added a new feature `gokart.testing.assert_frame_equal()` for comparing pd.DataFrame.

As mentioned in the issue, this is useful when index/columns orders are different.

Related issue: https://github.com/m3dev/gokart/issues/243  raised by @vaaaaanquish 

Please review!

### CODE REVIEW

File: gokart/testing/__init__.py [hunk 1]
```diff
@@ -1 +1,2 @@
 from gokart.testing.check_if_run_with_empty_data_frame import test_run, try_to_run_test_for_empty_data_frame
+from gokart.testing.pandas_assert import assert_frame_equal
```
Comment (hirosassa) [inline gokart/testing/__init__.py:2]: Please add new line at the end of this line.

File: gokart/testing/pandas_assert.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import pandas as pd
+
+
+def assert_frame_equal(actual: pd.DataFrame, expected: pd.DataFrame, **kwargs):
+    """
+    Assert that two DataFrames are equal.
+    Note
```
Comment (hirosassa) [inline gokart/testing/pandas_assert.py:7]: There's no contents of `Note`.

Comment (hirosassa) [reply gokart/testing/pandas_assert.py:7]: It needs a description of difference of `pd.testing.assert_frame_equal`, IMO

**Review by hirosassa — CHANGES_REQUESTED** [review]

**Review by vaaaaanquish — APPROVED** [review]
The most exciting PR in recent times.

**Review by Hi-king — CHANGES_REQUESTED** [review]
Need a explanation of difference from `pd.testing.assert_frame_equal`, plz

File: gokart/testing/__init__.py (hunk 1 above)
Comment (mski-iksm) [reply gokart/testing/__init__.py]: 👍

File: gokart/testing/pandas_assert.py (hunk 2 above)
Comment (mski-iksm) [reply gokart/testing/pandas_assert.py]: @hirosassa @Hi-king 
I've added a description of difference between pd.testing.assert_frame_equal.

Please re-review!

**Review by hirosassa — APPROVED** [review]
Thanks! LGTM

File: gokart/testing/pandas_assert.py (hunk 2 above)
Comment (hirosassa) [reply gokart/testing/pandas_assert.py]: 👍

**Comment by Hi-king** [comment]
@mski-iksm thx I've understand functionalities! 

IMO, how about more descriptive name indicating difference (e.g. assert_frame_contents_equal, assert_frame_equal_ignoring_index)

**Comment by mski-iksm** [comment]
@Hi-king 
> IMO, how about more descriptive name indicating difference (e.g. assert_frame_contents_equal, assert_frame_equal_ignoring_index)

I agree with your idea. I've changed the method name assert_frame_equal -> assert_frame_contents_equal .

Please review.

**Review by Hi-king — APPROVED** [review]

**Comment by Hi-king** [comment]
@mski-iksm 🆒 LGTM!

**Comment by Hi-king** [comment]
@mski-iksm Thx for cool function :) Merged!

**Pull request merged**